canonical-websites/www.ubuntu.com
# Use cookie policy package

## Done

- Replaced the cookie policy code with the cookie-policy npm package.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that the cookie policy is shown (you might need to use a private browser window).
